configure.ac

```diff
@@ -1,313 +1,127 @@
-#                                               -*- Autoconf -*-
-# Process this file with autoconf to produce a configure script.
+dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.59)
+
 AC_INIT(Normaliz, 3.7.4)
+
 AC_SUBST(LIBNORMALIZ_VERSION_MAJOR, 3)
 AC_SUBST(LIBNORMALIZ_VERSION_MINOR, 7)
 AC_SUBST(LIBNORMALIZ_VERSION_PATCH, 4)
 AC_SUBST(LIBNORMALIZ_VERSION_STRING, "$PACKAGE_VERSION")
-AM_INIT_AUTOMAKE(foreign)
```

Comment:
I really would suggest to keep the `foreign` here. Slavishly following the GNU conventions seems pointless. Of course it doesn't hurt to e.g. rename the change log file, but adding symlinks `README` and `NEWS` is simply silly (no offense intended).

Reply:
I like to follow this standard. If packagers find all the files that are usually around it makes their life easier as they do not have to think about how stuff might be named.

Reply:
I've worked as a package for 20 years, and you always have to check for which of these files exist and in which variant, and it is one of the easiest aspects of the whole process, so I really don't agree with this justification.

Reply:
I have no strong opinion on this. I just felt that this would streamline things a bit.

Reply:
Well, I find it outright confusing to have these pointless symlinks there. There is a reason why the "foreign" option exists, and why many (most?) non-GNU projects use it.